README.md

```diff
@@ -15,7 +15,7 @@ Set desired metas in your controller setup :
 
 ```js
 export default Ember.Route.extend({
-  setupController: function (controller, model) {
+  afterModel() {
```

Comment:
`setupController` is generally discouraged these days. I have seen several people having issues with it in the Ember slack channel. It also stops my model from being populated with data if I used `setupController`, so I think `afterModel` is the correct choice for the current Ember standards. Also, controllers are supposed to be going away at some point.